src/Event/BaseEvent.php

```diff
@@ -77,9 +77,9 @@ abstract public function getEventType();
     /**
      * Returns the array to be used for encoding this object as json.
      *
-     * @return array
+     * @return mixed
      */
-    public function jsonSerialize(): mixed
+    public function jsonSerialize()
```

Comment:
I don't think you can specify this return type in php 7.2 which this library is still supporting.